### DISCUSSION TITLE
Accidental solution submission

### DISCUSSION THREAD

**Discussion by Akilaydin** [codewars/codewars.com#3177] [Q&A] [answered]
Hello! I am writing to apologize for an unfortunate incident that occurred recently. I accidentally submitted a Python solution for a very difficult kata, which has resulted in my account being restricted. I would like to express my sincere apologies for any inconvenience this may have caused. I am not familiar with the Python language and did not intend to submit a solution written in that language. Instead, I was only interested in checking the edge case for my C# solution I had written myself. During the test run, I accidentally clicked "submit" for the Python solution I found online. I understand that this is not the appropriate action, but I didn't mean to stole the solution as if I did it myself. All I wanted is to run the tests on Python code and see if they pass in the place where my own C# solution fails. 
I would appreciate it if you could remove the account restriction and reverse my submission in Python. I assure you that I have no intention of submitting solutions that are not my own. 

I really like Codewars and I don't want to lose the honest progress I've made through the years.
Thank you for your attention.

**Comment by Blind4Basics** [comment, marked as answer]
Done. Note that just "attempting" gives you the reward. So _never_ try code you shouldn't in a trainer.

**Comment by jroycodes** [comment]
Didn't know that could happen? Has you account been unrestricted?
